### PR TITLE
Added a faq item for the container volume paths information

### PR DESCRIPTION
**Pull request by gunamata**
Signed-off-by: Gunasekhar Matamalam <gunasekhar.matamalam@suse.com>

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -167,3 +167,19 @@ echo "export PATH=\$PATH:/home/$(whoami)/.local/bin" >> ~/.bashrc
 - Copy "Rancher Desktop" shortcut from Desktop and paste in Startup folder.
 - Restart your machine.
 ```
+#### **Where does Rancher Desktop actually put the data volumes?**
```
Comment (mook-as) [inline docs/faq.md:170]:
```suggestion
#### **Q: Where does Rancher Desktop actually put the data volumes?**
```

File: docs/faq.md [hunk 2]
```diff
@@ -167,3 +167,19 @@ echo "export PATH=\$PATH:/home/$(whoami)/.local/bin" >> ~/.bashrc
...
+
+**A:** 
+
+**Windows:**
+Open Run menu (Press Windows + R) and open the path provided below, depending on the active container runtime.
+```
+dockerd(moby): \\wsl$\rancher-desktop-data\var\lib\docker\volumes
+containerd: \\wsl$\rancher-desktop-data\var\lib\nerdctl\dbb19c5e\volumes\<namespace>
```
Comment (mook-as) [inline docs/faq.md:178]: FWIW, [nerdctl docs](https://github.com/containerd/nerdctl/blob/b235a096085039ad4238ee5c70dbf6bfa11da76f/docs/dir.md#datarootaddrhash) explain where `dbb19c5e` comes from; I think that means it's going to be a constant for us.

File: docs/faq.md [hunk 3]
```diff
@@ -167,3 +167,19 @@ echo "export PATH=\$PATH:/home/$(whoami)/.local/bin" >> ~/.bashrc
...
+```
+**macOS & Linux:**
+Navigate to the path provided below in the VM(lima), depending on the active container runtime. You can use `rdctl shell` to access these paths in the VM.
```
Comment (mook-as) [inline docs/faq.md:181]:
```suggestion
Navigate to the path provided below in the VM, depending on the active container runtime. You can use `rdctl shell` to access these paths in the VM.
```

Alternatively, _… in the (lima) VM_.

**Review by mook-as — CHANGES_REQUESTED** [review]

File: docs/faq.md (hunk 1 above)
Comment (gunamata) [reply docs/faq.md:170]: Good catch. Thank you :).

File: docs/faq.md (hunk 3 above)
Comment (gunamata) [reply docs/faq.md]: Modified per your suggestion.

**Review by mook-as — APPROVED** [review]

**Pull request merged**